manifests/bucketeer/values.dev.yaml

```
@@ -73,6 +73,12 @@ backend:
       clientSecret:
       redirectUrls:
         - https://localhost:9000/auth/callback
+    password:
+      username: demo
+      password: demo
```

Let's pass the email, too.
So we won't need [this](https://github.com/bucketeer-io/bucketeer/pull/1121/files#diff-5eaf73ed7669ba7a0835bcd9ed3136f48691b71172b9db59f538e34cd046f676R29).

```suggestion
      password: demo
      email: demo@bucketeer.io
```